node/proxy/api/src/zrx.ts

```diff
@@ -0,0 +1,80 @@
+import axios, { isAxiosError } from 'axios'
+import { Axios } from 'axios'
+import { Request, Response } from 'express'
+
+const ZRX_API_KEY = process.env.ZRX_API_KEY
+
+if (!ZRX_API_KEY) throw new Error('ZRX_API_KEY env var not set')
+
+export class Zrx {
+  private axiosInstance: Axios
+
+  constructor() {
+    this.axiosInstance = axios.create({
+      headers: {
+        '0x-api-key': ZRX_API_KEY,
+      },
+    })
+  }
+
+  async handler(req: Request, res: Response): Promise<void> {
+    const parsedUrl = new URL('https://dummy.com'.concat(req.url))
+    const path = parsedUrl.pathname.substring('/api/v1/zrx/'.length)
```

Comment:
🧠

Reply:
Could potentially use a capture group to make this less leetcody magic but meh

Reply:
was following the pattern, though not sure how I ended up with substring at one point lol. I will just update with a simple `.replace()`